classes/Domain/Entity/Mapper/Talk.php

```diff
@@ -44,8 +44,18 @@ public function getAllPagerFormatted($admin_user_id, $options = [])
             ->with(['favorites', 'comments']);
         $formatted = [];
 
+        if ($where) {
```

Comment:
After merging, I had to make quite a few modifications to this code to get it to function as I expected it to. Specifically, the incomplete introduction of `$sort` (replacing `$options`) caused several notices that it was not properly initialized and a fatal error on calls to `Talk::getSortOptions`. I have resolved this and think that this PR is salvageable, but would advise that we look at cleaner ways of implementing these 4 export features. Michael has already said this work was pretty-much proof-of-concept to scratch an itch.